docs/section19/19-17.md

```diff
@@ -0,0 +1,58 @@
+- Check if B-RMR is modeled with at least one air-side system that is of type 9 or 10, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):` 
+    - Create dictionary of hvac systems and associated zones and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys_x = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`  
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Reset all_design_setpoints_105 boolean to true: `all_design_setpoints_105 = true`  
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `If baseline_system_type_compare(sys_type, HVAC_SYS.SYS_9, false) == TRUE OR baseline_system_type_compare(sys_type, HVAC_SYS.SYS_10, false) == TRUE: `    
+            - Get list of zones served by the hvac system: `zone_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]` 
+            - Get list of terminal units served by the hvac system: `terminal_list_hvac_sys_x = dict_of_zones_and_terminal_units_served_by_hvac_sys_x[hvac.id]["ZONE_LIST"]["Terminal_Unit_List"]`  
+            - For each terminal unit associated with the hvac system (should only be one): `for terminal in terminal_list_hvac_sys_x:`  
+                - Get the design supply temperature setpoint: `design_supply_air_temp_setpoint = terminal.supply_design_heating_setpoint_temperature`  
+                - Check if the design supply air temperature setpoint does not equal 105F, if it doesn't then set all_design_setpoints_105 to false: `if design_supply_air_temp_setpoint != 105: all_design_setpoints_105 = false`  
+            Below sums the terminal unit primary air flow associated with each zone that this baseline hvac system serves in the baseline in the proposed in order to compare baseline and proposed supply cfm in the Case 3 below:
+            - For each zone served by the HVAC system: `for zone in zone_list_hvac_sys_x:`  
+                - Get P_RMI zone object (however the RCT accomplishes this): `zone_p = match_data_element(P_RMI, Zone, zone.id)`  
+                    - For each terminal unit associated with the zone: `for terminal_p in zone_p.terminals:`  
+                        - Add the terminal air flow to the proposed_supply_cfm (I think the sum of terminal unit primary air flow shoould sum to the supply cfm of the hvac system serving all the terminals? Agreed? That is what this assumes): `proposed_supply_cfm = proposed_supply_cfm + terminal_p.primary_airflow`              
+            
```

Comment:
i think that is correct.  sum of primary air flow of all terminals should be equal to supply CFM of the HVAC system